### PR TITLE
use BG Ponder Indexer for fetching data

### PR DESCRIPTION
**Pull request by technophile-04**
## Completed : 

- [X] Fetching withdraw events data on `/projects` page
- [X] Fetching withdraw events data on `/members` page

## TODO : 

- [ ] Fetch builders data

### CODE REVIEW

**Comment by technophile-04** [comment]
@carletex , it seems that while fetching events for old contract 0x2eA63c9C9C114ae85b1027697A906420a23e8572 it only fetches 9 events where as total events are 32, here is the query I am running : 

```gql
query Withdrawls {
  cohortWithdrawals(
    where: {cohortContractAddress: "0x2eA63c9C9C114ae85b1027697A906420a23e8572"}
    orderBy: "timestamp"
    orderDirection: "desc"
  ) {
    reason
    builder
    amount
    timestamp
    cohortContractAddress
    builder
    id
  }
}
```

is it something to do with indexing? or maybe I am is there a field while making query that we can add to get all data?

File: packages/nextjs/pages/members.tsx [hunk 1]
```diff
@@ -26,27 +27,7 @@ const Members: NextPage = () => {
...
-  });
-
-  const sortedWithdrawEvents = useMemo(
-    () =>
-      [...(newContractWithdrawEvents || []), ...(oldContractWithdrawEvents || [])].sort(
-        (a: any, b: any) => b.block.number - a.block.number,
-      ),
-    [newContractWithdrawEvents, oldContractWithdrawEvents],
-  );
+  const { data: allWithdrawEvents, isLoading: isWithdrawEventsLoding } = useCohortWithdrawEvents();
```
Comment (damianmarti) [inline packages/nextjs/pages/members.tsx:30]: Typo isWithdrawEventsLoding -> isWithdrawEventsLoading

File: packages/nextjs/pages/projects.tsx [hunk 2]
```diff
@@ -56,27 +54,7 @@ const projects = [
...
-  });
-
-  const sortedWithdrawEvents = useMemo(
-    () =>
-      [...(newContractWithdrawEvents || []), ...(oldContractWithdrawEvents || [])].sort(
-        (a: any, b: any) => b.block.number - a.block.number,
-      ),
-    [newContractWithdrawEvents, oldContractWithdrawEvents],
-  );
+  const { data: allWithdrawEvents, isLoading: isWithdrawEventsLoding } = useCohortWithdrawEvents();
```
Comment (damianmarti) [inline packages/nextjs/pages/projects.tsx:57]: Same typo

File: packages/nextjs/pages/members.tsx (hunk 1 above)
Comment (carletex) [reply packages/nextjs/pages/members.tsx]: Thanks Damu, fixed!

**Comment by carletex** [comment]
We ended up recreating the Database, as suggested here: https://github.com/ponder-sh/ponder/issues/580#issuecomment-1897865260

Everything seems to be working now. 

Merging! Great job @technophile-04

**Pull request merged**